### DISCUSSION TITLE
Feature Request! Will be a Game Changing Apps

### DISCUSSION THREAD

**Discussion by mardisiaw** [AppGround-io/bluetooth-keyboard-and-mouse-support#278] [Ideas]
Why dont the apps let the user to choose their own images for custom button in custom layout? Animated GIF will be a game changing.

This is an example how the apps could be a limitless & creative input device.
![giphy](https://user-images.githubusercontent.com/7744159/207663312-5724b422-c5ea-4c5a-b90d-3fcf9e8c761e.gif)

**Comment by ExperiencesStudio** [comment]
It now supports the basic emojis, but if it supports custom emojis, that can be user added button icon it can go mainstream in things like editing or 3d. For now i try to use the ones from the default menu by association, but it is hard to learn when using program with many specific tools, and text description is large and not intuitive.
